build_project/local.go

```diff
@@ -0,0 +1,66 @@
+package build
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/tarantool/cartridge-cli/common"
+	"github.com/tarantool/cartridge-cli/project"
+)
+
+func buildProjectLocally(projectCtx project.ProjectCtx) error {
+	// pre-build
+	preBuildHookPath := filepath.Join(projectCtx.BuildDir, preBuildHook)
+
+	if _, err := os.Stat(preBuildHookPath); !os.IsNotExist(err) {
+		err = runHook(preBuildHookPath, projectCtx.BuildDir)
+		if err != nil {
+			return fmt.Errorf("Failed to run pre-build hook: %s", err)
+		}
+	}
+
+	// tarantoolctl rocks make
+	if rockspec, err := common.FindRockspec(projectCtx.BuildDir); err != nil {
+		return err
+	} else if rockspec != "" {
+		rocksMakeCmd := exec.Command("tarantoolctl", "rocks", "make")
```

Comment:
offtop: it's  quite ironic. We rewrite it in go to have independent binary file. Which in fact strictly depends on "tarantoolctl" which depends on "tarantool".

Reply:
Seems that before run build (in fact each command) we should validate that such tools are available.

tarantoolctl, git, etc... I think user should know about dependency problem before we will do anything

Reply:
Yes, but `tarantool` isn't required for creating a project, or building in docker....

Reply:
Yes. It doesn't contradict with the fact that we should check tarantool existence as early as possible when it needed.